vnext/Desktop/DevSupportManager.cpp

```diff
@@ -58,27 +58,33 @@ string DevSupportManager::GetJavaScriptFromServer(
 
   Url url(bundleUrl);
   auto const resolveResult = m_resolver.resolve(url.host, url.port);
-  tcp::socket socket{m_context};
-  connect(socket, resolveResult);
+  tcp_stream stream{m_context};
 
-  request<string_body> request{verb::get, url.Target(), 11};
-  request.set(field::host, url.host);
-  request.set(field::user_agent, BOOST_BEAST_VERSION_STRING);
+  stream.connect(resolveResult);
 
-  write(socket, request);
+  http::request<http::string_body> request{http::verb::get, url.Target(), 11};
```

Comment:
>11 [](start = 74, length = 2)

(nit) can you either add comment why it's 11 or better create some const

Reply:
It refers to HTTP 1.1.
I'll add the comment.